pkgs/development/libraries/libraw1394/default.nix

```diff
@@ -11,6 +11,7 @@ stdenv.mkDerivation rec {
   meta = { 
     description = "Library providing direct access to the IEEE 1394 bus through the Linux 1394 subsystem's raw1394 user space interface";
     homepage = "https://ieee1394.wiki.kernel.org/index.php/Libraries#libraw1394";
-    license = ["GPL" "LGPL"];
+    license = stdenv.lib.licenses.lgpl2;
```

Comment:
I see now: the files generated by autotools are listed by Debian as licensed under GPL, because autotools is GPL. However this might not be quite right, because there's a GPL exception for that particular case https://www.gnu.org/licenses/autoconf-exception-3.0.html. Anyway, nobody's going to have any fun trying to figure out this legal garbage, so I'm happy to resolve this in whatever way the two of you think is most appropriate. Just let me know.